project/gh-action/infrastructure/kubernetes/deployment.yaml

```
@@ -0,0 +1,58 @@
+          periodSeconds: 10
+        livenessProbe:
+          httpGet:
+            path: /health
+            port: http
+          initialDelaySeconds: 15
+          periodSeconds: 20
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+              - ALL
```

yamllint (1.35.1)

[warning] wrong indentation: expected 12 but found 14

(indentation)